DiscordBot/Program.cs

```diff
@@ -40,9 +47,57 @@ private static async Task MainAsync()
             commands.RegisterCommands<EventCommands>();
 
             discord.UseInteractivity(new InteractivityConfiguration());
+            discord.MessageReactionAdded += OnMessageReactionAdded;
 
             await discord.ConnectAsync();
             await Task.Delay(-1);
         }
+
+        private static async Task OnMessageReactionAdded(
+            DiscordClient client,
+            MessageReactionAddEventArgs eventArguments)
+        {
+            if (eventArguments.User == client.CurrentUser)
```

Comment:
What's this case for? 🙂

Reply:
client.CurrentUser is the bot itself so this is the bot ignoring the reactions that are initialised on the signup sheet.

Reply:
Ah okay, might be worth a comment explaining that 🙂